Gemfile

```diff
@@ -12,6 +12,9 @@ gem 'rspec', '~> 3'
 gem 'rspec_junit_formatter', '~> 0.6'
 
 gem 'codecov', '~> 0.6', require: false
+gem 'debug', '~> 1.9', require: false
+gem 'racc', '~> 1.7', require: false
+gem 'retest', '~> 1.13', require: false
```

Comment:
For transparency:
- `debug` isn't necessary, it's just a nice alternative/complement to `pry`
- `racc` was required for me to run `rubocop` (Ruby 3.3.0, Mac running Sonoma)
- [`retest`](https://github.com/AlexB52/retest) is just super helpful, essentially opinionated `guard` without a Guardfile, and it supports `focus` out of the box 😍